docs/getting-started.md

```diff
@@ -7,18 +7,22 @@ The next examples give you a simple skeleton on how to use the KILT SDK to creat
 
 - [1. How to install the SDK](#1-how-to-install-the-sdk)
   - [1.1. Prerequisites](#11-prerequisites)
+  - [1.2. Connect to a Chain](#11-connect-to-a-chain)
 - [2. How to generate an Identity](#2-how-to-generate-an-identity)
-- [3. How to build and store a Claim Type (CTYPE)](#3-how-to-build-and-store-a-claim-type-ctype)
+  - [2.1. Generate a Keystore](#21-generate-a-keystore)
+  - [2.2. Generate a light DID for the Claimer](#22-generate-a-light-did-for-the-claimer)
+  - [2.3. Generate a full DID for the Attester](#22-generate-a-full-did-for-the-attester)
+* [3. How to build and store a Claim Type (CTYPE)](#3-how-to-build-and-store-a-claim-type-ctype)
```

Comment:
Should we just call it `Build and store...`? Not beginning with how, as it is not used anywhere else (if we also change at point 4 and 5).

Reply:
I have updated all the places and removed the `how to`